solidity-v1/dashboard/src/lib/keep/index.js

```diff
@@ -199,10 +207,26 @@ class Keep {
     this.keepToTStaking = new KeepToTStaking(
       this.thresholdStakingContract,
       this.thresholdKeepStakeContract,
+      this.simplePREApplicationContract,
       this.web3
     )
   }
 
+  _initializePREContract = async (web3) => {
+    const chainId = await web3.lib.eth.getChainId()
```

Comment:
We can get chain id using `getChainId` fn in `src/connectors/utils`. Or let's update the keep lib a bit and add chain id to the `Keep` constructor so we make the lib independent of dapp implementation.

Reply:
a3ab04267c9d6acc69725ef3e469c8817810ac8e

Reply:
Changed a little bit in 42f1b04dc22da70f4cd4b1ac507181aeb94ae941